modules/ossm-troubleshooting-proxy.adoc

```diff
@@ -0,0 +1,47 @@
+// Module included in the following assemblies:
+// * service_mesh/v2x/-ossm-troubleshooting-istio.adoc
+
+[id="ossm-troubleshooting-proxy_{context}"]
+= Troubleshooting Envoy proxy
+
+The Envoy proxy intercepts all inbound and outbound traffic for all services in the service mesh. Envoy also collects and reports telemetry on the service mesh. Envoy is deployed as a sidecar to the relevant service in the same pod.
+
+== Enabling access logs
+
+Envoy access logs useful in diagnosing traffic failures and flows and help with end-to-end traffic flow analysis.
+
+To enable access logging for all istio-proxy containers, edit the ServiceMeshControlPlane (SMCP) file.
+
+[source,terminal]
+----
+$ oc edit smcp <SMCP-name>
+----
+
+[source,yaml]
+----
+spec:
+  proxy:
+    accessLogging:
+      file:
+        name: /dev/stdout     #file name
+----
+
+
+== Common Envoy error messages
+`Error message:
+2020-08-22T15:06:29.105313Z     info    Envoy proxy is NOT ready: config not received from Pilot (is Pilot running?): cds updates: 1 successful, 0 rejected; lds updates: 0 successful, 0 rejected`
+
+Possible Causes
+
+* Rejected Config
+* Envoy log: gRPC config for type.googleapis.com/... rejected
```

Comment:
Are you expecting to fill these out more? The log messages don't really tell me why it was rejected, just things to look for.

Reply:
Yeah, as we collect more troubleshooting information, we're going to keep adding.  This is just to get something published that we can build on.

Reply:
I think Brian's comment is key here. I think the other modules in this PR are a good start to the troubleshooting assembly that we can add to. But, I ​don't think this module is ready for publishing yet. 

I think the possible causes needs more explanation around what the possible causes are and what a user can do if those possible causes happen in their instance.

Reply:
I pulled the part about error messages until it's more developed, but left in the content about how to enable the logs.